src/utils/api/makeApiUrl.ts

```diff
@@ -1,4 +1,4 @@
-export const makeApiUrl = (key) =>
```

Comment:
It's the same as `src/firebase/utils/makeApiUrl.ts` so I think one of them could be removed. The folder `api` with its contents could be moved out of `utils` directly inside `src`.